crates/sui/src/keytool.rs

```diff
@@ -741,6 +766,26 @@ impl KeyToolCommand {
                 CommandOutput::Show(key)
             }
 
+            KeyToolCommand::ZkLoginSignPersonalMessage { data } => {
+                let msg = PersonalMessage {
+                    message: data.as_bytes().to_vec(),
+                };
+                let intent_msg = IntentMessage::new(Intent::personal_message(), msg.clone());
+
+                let skp =
+                    SuiKeyPair::Ed25519(Ed25519KeyPair::generate(&mut StdRng::from_seed([0; 32])));
```

Comment:
Seems weird to hard-code eph key pair, max epoch in `ZkLoginSignPersonalMessage` which sounds like a feature that should allow signing arbitrary messages. Why are we doing this? Is it only for debugging purposes?

Reply:
this is for testing purpose. otherwise i need to ask the user to go through a login flow instead to get valid jwt and get valid proof (instead of using a deterministic zklogin inputs). i dont see anyone asks for a "real" keytool here so using the fixed one.

Reply:
Could you add a comment to indicate that this is meant only for testing purpose?

Reply:
added. note that with proof domain separated as dev use only and max epoch fixed to 10, it is not possible for misuse in anything not devnet before epoch 10.